EpiResponsivePicture/Blob/BlobImageProvider.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Baaijte.Optimizely.ImageSharp.Web.Resolvers;
+using EPiServer;
+using EPiServer.Core;
+using EPiServer.Web.Routing;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using SixLabors.ImageSharp.Web;
+using SixLabors.ImageSharp.Web.Providers;
+using SixLabors.ImageSharp.Web.Resolvers;
+
+namespace Forte.EpiResponsivePicture.Blob;
+
+public class BlobImageProvider : IImageProvider
```

Comment:
I wonder if it's not better to extend `Baaijte.Optimizely.ImageSharp.Web.Providers.BlobImageProvider` and override `Match` method (and of course add all stuff like AddSegments etc)? This class is tightly coupled with Baajite implementation anyway (due to `BlobImageResolver` usage) so we can avoid rewriting GetAsync, ProcessingBehavior etc.
What do you think?

Reply:
You're right. But I changed it even better I think. Using original `BlobImageResolver` from `Baaijte.Optimizely.ImageSharp.Web.Providers` - figured out that `IsMatch` has public setter, so I can assign custom functionality into it.